src/datatype.ts

```diff
@@ -12,6 +12,20 @@ import {ISelectAble, SelectAble} from './idtype';
 import {all, none, Range1D, Range1DGroup, composite, Range} from './range';
 
 /**
+ * TODO
+ */
+export interface IDataDescriptionValue {
```

Comment:
@thinkh: I'm not invested in this. If this will be better handled by https://github.com/Caleydo/caleydo_web_container/issues/34, that's fine with me.